ui/raidboss/data/06-ew/dungeon/another_aloalo_island.ts

```diff
@@ -0,0 +1,1550 @@
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'AAI Snipper Bubble Shower',
+      type: 'StartsUsing',
+      netRegex: { id: '8BB9', source: 'Aloalo Snipper', capture: false },
+      response: Responses.goSides(),
+    },
+    {
+      id: 'AAI Snipper Crab Dribble',
+      type: 'Ability',
+      // Crab Dribble 8BBA has a fast cast, so trigger on Bubble Shower ability
+      netRegex: { id: '8BB9', source: 'Aloalo Snipper', capture: false },
+      response: Responses.goFront('info'),
+    },
+    {
+      id: 'AAI Ray Hydrocannon',
+      type: 'StartsUsing',
+      netRegex: { id: '8BBD', source: 'Aloalo Ray', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'AAI Ray Expulsion',
+      type: 'StartsUsing',
+      netRegex: { id: '8BBF', source: 'Aloalo Ray', capture: false },
+      response: Responses.getOut(),
+    },
+    {
+      id: 'AAI Ray Electric Whorl',
+      type: 'StartsUsing',
+      netRegex: { id: '8BBE', source: 'Aloalo Ray', capture: false },
+      response: Responses.getUnder('alert'),
+    },
+    {
+      id: 'AAI Monk Hydroshot',
+      type: 'StartsUsing',
+      netRegex: { id: '8C65', source: 'Aloalo Monk' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.knockbackOn(),
+    },
+    {
+      id: 'AAI Monk Cross Attack',
+      type: 'StartsUsing',
+      netRegex: { id: '8BBB', source: 'Aloalo Monk' },
+      response: Responses.tankBuster(),
+    },
+    // ---------------- Ketuduke ----------------
+    {
+      id: 'AAI Ketuduke Tidal Roar',
+      type: 'StartsUsing',
+      netRegex: { id: '8AD4', source: 'Ketuduke', capture: false },
+      response: Responses.bleedAoe(),
+    },
+    {
+      id: 'AAI Ketuduke Bubble Net',
+      type: 'StartsUsing',
+      netRegex: { id: '8AAD', source: 'Ketuduke', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'AAI Ketuduke Spring Crystals',
+      type: 'StartsUsing',
+      netRegex: { id: '8AA8', source: 'Ketuduke', capture: false },
+      run: (data) => {
+        data.ketuSpringCrystalCount++;
+        data.ketuCrystalAdd = [];
+      },
+    },
+    {
+      id: 'AAI Ketuduke Spring Crystal 2 Collect',
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '12607' },
+      run: (data, matches) => data.ketuCrystalAdd.push(matches),
+    },
+    {
+      id: 'AAI Ketuduke Bubble Weave/Foamy Fetters',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['E9F', 'ECC'] },
+      infoText: (data, matches, output) => {
+        if (data.me !== matches.target)
+          return;
+        data.ketuMyBubbleFetters = matches.effectId;
+        if (matches.effectId === 'E9F')
+          return output.bubble!();
+        return output.bind!();
+      },
+      run: (data, matches) => data.gainList.push(matches),
+      outputStrings: {
+        bubble: {
+          en: 'Bubble',
+          ja: 'バブル',
+        },
+        bind: {
+          en: 'Bind',
```

Comment:
It might be better to use "Fetters" here since it matches the in-game debuff name better and also sounds more audibly different from "Bubble" than "Bind".